cmd/flags/target.go

```diff
@@ -43,8 +45,10 @@ func (t *Target) String() string {
 func (t *Target) initFlags() {
 	flag.StringVar(&t.HTTPHost, "target-http-host", "http://localhost", "HTTP host to warm up")
 	flag.IntVar(&t.HTTPPort, "target-http-port", 8080, "HTTP port for warm up requests")
+	flag.IntVar(&t.HTTPTimeoutMilliseconds, "target-http-timeout-milliseconds", 10000, "HTTP timeout for requests")
```

Comment:
default same as previous hardcoded value. note that the previous one was in seconds.